cyclist/src/edu/utexas/cycic/Cycic.java

```diff
@@ -540,15 +539,8 @@ public void buildDnDCircle(FacilityCircle circle, int i, String name){
 		circle.setFill(Color.rgb(circle.rgbColor.get(0),circle.rgbColor.get(1),circle.rgbColor.get(2)));
 		circle.setCenterX(45+(i*90));
 		circle.setCenterY(50);
-		circle.text.setText(name.split(" ")[2]);
-		circle.text.setWrapText(true);
-		circle.text.setMaxWidth(60);
-		circle.text.setLayoutX(circle.getCenterX()-circle.getRadius()*0.7);
-		circle.text.setLayoutY(circle.getCenterY()-circle.getRadius()*0.6);	
-		circle.text.setTextAlignment(TextAlignment.CENTER);
-		circle.text.setMouseTransparent(true);
-		circle.text.setMaxWidth(circle.getRadius()*1.4);
-		circle.text.setMaxHeight(circle.getRadius()*1.2);
+		circle.text.setText(name.split(" ")[2] + " (" + name.split(" ")[1] + ")");
+		VisFunctions.placeTextOnCircle(circle,"middle");
```

Comment:
Right now, this function reads the size of the circle and place text based on that, so as long as the skin still has an underlying radius, this can be managed by setting that size to be correct.  Is that there?